scalastyle-config.xml

```diff
@@ -250,6 +250,14 @@ This file is divided into 3 sections:
     <customMessage>Omit braces in case clauses.</customMessage>
   </check>
 
+  <!-- SPARK-16877: Avoid Java annotations -->
+  <check customId="OverrideJavaCase" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
+    <parameters><parameter name="regex">^Override$</parameter></parameters>
```

Comment:
Hm so this matches "@Override" but not "Override" now, but would reverse if the regex included "@"? That sounds flipped. @ isn't a special char. You're doubly sure that's right? It also seems like this is matching "Override" on a line alone but should be looking for "@Override" anywhere.